@codex please fix

```
> dspace@3.0.0 format:check
> prettier --config .prettierrc --check "**/*.{js,ts,svelte,json,md}"

Checking formatting...
 ✓ tests/eslintPluginPresence.test.ts (1 test) 6ms
 ✓ frontend/tests/gameStateFlush.test.ts (1 test) 9ms
 ✓ tests/checkoutFetchDepth.test.ts (1 test) 10ms
 ✓ tests/npmTestRunsAllSuites.test.ts (1 test) 6ms
 ✓ tests/astroConfigLogLevel.test.ts (1 test) 4ms
 ✓ tests/prismSvelte.test.ts (1 test) 17ms
 ✓ scripts/tests/dbBenchmarkScript.test.ts (1 test) 4ms
 ✓ tests/progressionBalance.test.ts (1 test) 32ms
 ✓ tests/domException.test.ts (1 test) 3ms
 ✓ tests/indexeddb.server.test.ts (1 test) 308ms
   ✓ getItems returns empty array when IndexedDB unsupported  306ms
 ✓ frontend/tests/sanity.test.ts (1 test) 4ms
[warn] src/pages/processes/ProcessListRow.svelte
[warn] Code style issues found in the above file. Run Prettier with --write to fix.
 ❯ scripts/tests/prettierFormatting.test.ts (1 test | 1 failed) 19104ms
   × Prettier formatting > frontend sources are formatted 19101ms
     → Command failed: npm run format:check

⎯⎯⎯⎯⎯⎯⎯ Failed Tests 1 ⎯⎯⎯⎯⎯⎯⎯

 FAIL  scripts/tests/prettierFormatting.test.ts > Prettier formatting > frontend sources are formatted
Error: Command failed: npm run format:check
 ❯ scripts/tests/prettierFormatting.test.ts:8:5
      6|   test('frontend sources are formatted', () => {
      7|     const frontendDir = path.join(__dirname, '../../frontend');
      8|     execSync('npm run format:check', {
       |     ^
      9|       cwd: frontendDir,
     10|       stdio: 'inherit',

⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯[1/1]⎯


 Test Files  1 failed | 290 passed (291)
      Tests  1 failed | 1025 passed (1026)
   Start at  06:40:41
   Duration  165.72s (transform 9.27s, setup 10.82s, collect 44.79s, tests 109.34s, environment 188.85s, prepare 35.49s)


Error: Error: Command failed: npm run format:check
 ❯ scripts/tests/prettierFormatting.test.ts:8:5

⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯
Serialized Error: { status: 1, signal: null, output: [ null, null, null ], pid: 9411, stdout: null, stderr: null }

 ELIFECYCLE  Command failed with exit code 1.
Error: Process completed with exit code 1.
```

new codex task, not a r/e/v/i/e/w task

--- a/.github/workflows/ci-image.yml
+++ b/.github/workflows/ci-image.yml
@@ -45,10 +45,11 @@ jobs:
         with:
           node-version-file: .nvmrc
 
-      - name: Enable pnpm via Corepack
-        run: |
-          corepack enable
-          corepack prepare pnpm@9 --activate
+      - name: Setup pnpm
+        uses: pnpm/action-setup@v4
+        with:
+          version: 9.0.0
+          run_install: false
 
       - name: Verify pnpm is on PATH
         run: |
@@ -283,10 +284,11 @@ jobs:
         with:
           node-version-file: .nvmrc
 
-      - name: Enable pnpm via Corepack
-        run: |
-          corepack enable
-          corepack prepare pnpm@9 --activate
+      - name: Setup pnpm
+        uses: pnpm/action-setup@v4
+        with:
+          version: 9.0.0
+          run_install: false
 
       - name: Verify pnpm is on PATH
         run: |

--- a/.github/workflows/ci-sentinel.yml
+++ b/.github/workflows/ci-sentinel.yml
@@ -12,10 +12,11 @@ jobs:
         uses: actions/setup-node@v4
         with:
           node-version-file: '.nvmrc'
-      - name: Enable pnpm via Corepack
-        run: |
-          corepack enable
-          corepack prepare pnpm@9 --activate
+      - name: Setup pnpm
+        uses: pnpm/action-setup@v4
+        with:
+          version: 9.0.0
+          run_install: false
       - name: Install dependencies
         run: npm run ci:install
       - name: Validate workflow contracts

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -15,10 +15,11 @@ jobs:
       - uses: actions/setup-node@v4
         with:
           node-version: 20
-      - name: Enable pnpm via Corepack
-        run: |
-          corepack enable
-          corepack prepare pnpm@9 --activate
+      - name: Setup pnpm
+        uses: pnpm/action-setup@v4
+        with:
+          version: 9.0.0
+          run_install: false
       - name: Determine pnpm store path
         id: pnpm-store
         shell: bash
@@ -63,10 +64,11 @@ jobs:
         uses: actions/setup-node@v4
         with:
           node-version: 20
-      - name: Enable pnpm via Corepack
-        run: |
-          corepack enable
-          corepack prepare pnpm@9 --activate
+      - name: Setup pnpm
+        uses: pnpm/action-setup@v4
+        with:
+          version: 9.0.0
+          run_install: false
       - name: Determine pnpm store path
         id: pnpm-store
         shell: bash
@@ -93,10 +95,11 @@ jobs:
         uses: actions/setup-node@v4
         with:
           node-version: 20
-      - name: Enable pnpm via Corepack
-        run: |
-          corepack enable
-          corepack prepare pnpm@9 --activate
+      - name: Setup pnpm
+        uses: pnpm/action-setup@v4
+        with:
+          version: 9.0.0
+          run_install: false
       - name: Install dependencies
         run: pnpm install --frozen-lockfile --reporter=append-only
       - name: Setup Playwright environment

--- a/.github/workflows/quest-chart.yml
+++ b/.github/workflows/quest-chart.yml
@@ -12,10 +12,11 @@ jobs:
       - uses: actions/setup-node@v4
         with:
           node-version: 20
-      - name: Enable pnpm via Corepack
-        run: |
-          corepack enable
-          corepack prepare pnpm@9 --activate
+      - name: Setup pnpm
+        uses: pnpm/action-setup@v4
+        with:
+          version: 9.0.0
+          run_install: false
       - name: Determine pnpm store path
         id: pnpm-store
         shell: bash

--- a/.github/workflows/tests.yml
+++ b/.github/workflows/tests.yml
@@ -21,10 +21,11 @@ jobs:
         with:
           node-version: '20'
 
-      - name: Enable pnpm via Corepack
-        run: |
-          corepack enable
-          corepack prepare pnpm@9 --activate
+      - name: Setup pnpm
+        uses: pnpm/action-setup@v4
+        with:
+          version: 9.0.0
+          run_install: false
 
       - name: Determine pnpm store path
         id: pnpm-store
@@ -56,10 +57,11 @@ jobs:
         with:
           node-version: '20'
 
-      - name: Enable pnpm via Corepack
-        run: |
-          corepack enable
-          corepack prepare pnpm@9 --activate
+      - name: Setup pnpm
+        uses: pnpm/action-setup@v4
+        with:
+          version: 9.0.0
+          run_install: false
 
       - name: Determine pnpm store path
         id: pnpm-store
@@ -104,10 +106,11 @@ jobs:
         with:
           node-version: '20'
 
-      - name: Enable pnpm via Corepack
-        run: |
-          corepack enable
-          corepack prepare pnpm@9 --activate
+      - name: Setup pnpm
+        uses: pnpm/action-setup@v4
+        with:
+          version: 9.0.0
+          run_install: false
 
       - name: Determine pnpm store path
         id: pnpm-store

--- a/frontend/src/pages/processes/ProcessListRow.svelte
+++ b/frontend/src/pages/processes/ProcessListRow.svelte
@@ -39,18 +39,9 @@
                   .map((entry) => toPreviewLine(entry, metadataMap))
             : [];
 
-    $: requirePreviewLines = getPreviewLines(
-        process?.requirePreviewEntries,
-        itemMetadataMap
-    );
-    $: consumePreviewLines = getPreviewLines(
-        process?.consumePreviewEntries,
-        itemMetadataMap
-    );
-    $: createPreviewLines = getPreviewLines(
-        process?.createPreviewEntries,
-        itemMetadataMap
-    );
+    $: requirePreviewLines = getPreviewLines(process?.requirePreviewEntries, itemMetadataMap);
+    $: consumePreviewLines = getPreviewLines(process?.consumePreviewEntries, itemMetadataMap);
+    $: createPreviewLines = getPreviewLines(process?.createPreviewEntries, itemMetadataMap);
 </script>
 
 <article class="process-row" data-process-id={processId}>

